code/web/src/modules/user/Profile.js

```diff
@@ -9,54 +9,233 @@ import { Link } from 'react-router-dom'
+            <p style={{ color: grey2, marginBottom: '2em' }}>Email Address: {this.props.user.details.email}</p>
+            <p style={{ color: grey2, marginBottom: '2em' }}>Shipping Address: 123 Main Street, Denver, CO 80602</p>
+
+          </GridCell>
+        </Grid>
+
+        {/* Buttons */}
+        <Grid>
+          <GridCell style={{ flex: '' , textAlign: 'center', marginBottom: '2em' }}>
+            <Button theme="secondary" onClick={() => console.log('wow!')} style={{ marginRight: '1em' }}>Update Profile</Button>
+
+            <Link to={userRoutes.subscriptions.path}>
+              <Button theme="primary">Subscriptions</Button>
+            </Link>
+
+            <Button theme="secondary" onClick={this.props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
+          </GridCell>
+        </Grid>
+
+        {/* Bottom title bar */}
+        <Grid style={{ backgroundColor: grey }}>
+          <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+            <H3 font="secondary">My orders</H3>
+          </GridCell>
+        </Grid>
+
+        {/* Current orders */}
+        <Grid>
+          <GridCell>
+            {/* Card for each subscription crate - map through each in list array */}
+            <H4 font='primary' style={{ margin: '2rem' }}>Current Orders</H4>
+            <Grid>
+              <GridCell>
+                {this.props.subscriptions &&
+                  <Card style={{ width: '18em', backgroundColor: white, margin: 'auto' }}>
+                    <p style={{ padding: '2em 3em 0 3em' }}>
+                      <img src={`${ APP_URL }/images/crate.png`} alt={ this.props.subscriptions.crate.name } style={{ width: '100%' }}/>
+                    </p>
+
+                    <div style={{ padding: '1em 1.2em' }}>
+                      <H4 font="secondary" style={{ color: black }}>{ this.props.subscriptions.crate.name }</H4>
+
+                      <p>Order Number: {this.props.subscriptions.createdAt}</p>
+                      {/* shipping address confirmed if user.shippingAddress has value */}
+                      <p>Shipping Address: Confirmed</p>
+                      <p>Items in Crate: 3</p>
+                      <p>Delivery Date: </p>
+
+                      <p style={{ textAlign: 'center', marginTop: '1.5em', marginBottom: '1em' }}>
+                        <Button
+                          theme="secondary"
+                          onClick={() => console.log('Update date')}
+                          type="button"
+                          >
+                          Update Shipping Date
+                        </Button>
+                      </p>
+                    </div>
+                  </Card>
+                }
+              </GridCell>
+              <GridCell>
```

Comment:
Definitely looking forward to breaking these sections of code into their own components. I think that will significantly reduce the length of this file.